docs/analyzer.md

```diff
@@ -0,0 +1,24 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library
+that cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
Should we be pre-empting word wrapping, or should we let readthedocs/the user's browser handle it?